ecs-agent/csiclient/csi_client.go

```diff
@@ -29,6 +31,10 @@ import (
 const (
 	protocol        = "unix"
 	fsTypeBlockName = "block"
+
+	DefaultImageName      = "ebs-csi-driver"
+	DefaultSocketName     = "csi-driver.sock"
+	DefaultSocketHostPath = "/var/run/ecs/"
```

Comment:
we might need linux/windows specific versions of at least the default socket host path.  Thoughts?

Reply:
Right.... Let me split into Linux/Windows code paths. Good catch!

Reply:
Updated.